Debian packaging on debian10:

I found 1 failure and 1 error:
* there is a  test on locales and hence locales should be installed as Build-dep for debian packaging
* POCL which is still providing the wrong answer. This is still related to: https://github.com/pocl/pocl/issues/617

```
======================================================================
ERROR: test_locale_user (silx.io.test.test_specfile.TestSFLocale)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/user/workspace/silx/build/lib.linux-x86_64-3.6/silx/io/test/test_specfile.py", line 408, in test_locale_user
    locale.setlocale(locale.LC_NUMERIC, '')  # use user's preferred locale
  File "/usr/lib/python3.6/locale.py", line 598, in setlocale
    return _setlocale(category, locale)
locale.Error: unsupported locale setting

======================================================================
FAIL: test_medfilt (silx.opencl.test.test_medfilt.TestMedianFilter)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/user/workspace/silx/build/lib.linux-x86_64-3.6/silx/opencl/test/test_medfilt.py", line 115, in test_medfilt
    self.assertEqual(r.error, 0, 'Results are correct')
AssertionError: 3.4028235e+38 != 0 : Results are correct
```

This fails regardless if it is python3.6 or python3.7.

--- a/package/debian9/control
+++ b/package/debian9/control
@@ -44,7 +44,8 @@ Build-Depends: cython,
                python3-sphinx,
                python3-sphinxcontrib.programoutput,
                openstack-pkg-tools,
-               help2man
+               help2man,
+               locales
 Standards-Version: 3.9.8
 Vcs-Browser: https://anonscm.debian.org/cgit/debian-science/packages/silx.git
 Vcs-Git: git://anonscm.debian.org/debian-science/packages/silx.git

